### ISSUE TITLE
feature request: specify expected barcodes

### ISSUE THREAD

**Issue by jvolkening** [rrwick/Porechop#42]
For use in automated workflows, it would be nice to be able to specify expected barcodes on the command line to be included in output. For instance, if a user knows they only have libraries tagged with 'BC01' and 'BC02', they don't expect and don't care about reads (mis)classified into the 'BC06' bin. I'm imagining something like:
```
porechop --include_barcodes BC01,BC02 [...]
```

where any bins not on the inclusion list would be reported but not actually included in output. Of course, this can be dealt with using a downstream filter, but it seems this is logical enough to possibly include in Porechop itself.

**Comment by rrwick** [comment]
Yes, that's a good one. I'd more generally like to add the feature where any adapter or kit can be specified on the command line. When these are used, Porechop can skip the scan where it identifies which barcodes are present.

**Comment by Psy-Fer** [comment]
That would also solve the issue I just posted about requiring randomised fastq files.
